src/components/auth/SignInScreen.js

```diff
@@ -103,104 +111,140 @@ export default class SignInScreen extends React.Component {
                       onChangeText={value => this.onChangeText('password', value)}
                     />
                   </Item>
+                  {/* Sign Up Text */}
+                  <Text style={styles.buttonText2}>Novo usuário?{" "}
+                    <Text 
+                      onPress={() => this.props.navigation.navigate('SignUp')}
+                      style={styles.buttonText2link}>Crie uma conta 
+                    </Text>
+                    {" "}
+                    <MaterialCommunityIcons name = 'chevron-double-right'/>
+                  </Text>
+                  {/* Forgot Password Text */}
+                  <Text style={styles.buttonText2}>Esqueceu sua senha?{" "}
+                    <Text
+                      onPress={() => this.props.navigation.navigate('ForgotPassword')}
+                      style={styles.buttonText2link}>Recupere aqui
+                    </Text>
+                    {" "}
+                    <MaterialCommunityIcons name = 'chevron-double-right'/>
+                  </Text>
+                  {/* Loading ActivityIndicator */}
+                </View>  
+                <KeyboardAvoidingView style={styles.container2} behavior='position' enabled>
```

Comment:
Is it possible to move all this down just a bit when keyboard pops up?
![image](https://user-images.githubusercontent.com/5898380/87582732-afc88b00-c68f-11ea-8cd7-933ab3082422.png)